external_apps/common/router.go

```diff
@@ -46,4 +46,4 @@ func NewRouter() ControllerRouter {
 	}
 }
 
-var _ Router = &ControllerRouter{}
+var _ Router = &ControllerRouter{}
```

Comment:
maybe change the name of this router to something like controllerRouter. It will get confusing with all the routers.

Reply:
yeah sure